PhysicsTools/NanoAOD/python/nanoDQM_cfi.py

```diff
@@ -132,6 +132,32 @@
                 NoPlot('vidNestedWPBitmap'),
                 NoPlot('vidNestedWPBitmap_Fall17V2'),
                 NoPlot('vidNestedWPBitmapHEEP'),
+                Plot1D('ipLength', 'ipLength', 25, -0.25, 0.25, 'signed lenght of impact parameter (3d)'),
```

Comment:
it seems a bit awkward to have the plot defined in mainstream configuration, please implement those in the customise, assuming that one would run the NANO and DQM step in the same process

Reply:
OK.